src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -264,8 +319,6 @@ def launch_fluent(
     """
     if version != "3d" and gpu:
         raise GPUSolverSupportError()
-    _process_kwargs(kwargs)
```

Comment:
The aim in this function should be to remove all the extra code prior to calling `create_launcher()`. Then we can guarantee that create_launcher() works independently and that each launcher type works independently.

Reply:
Yes we will address this issue in next PR. Thank you.